xtras/src/send_next.rs

```diff
@@ -0,0 +1,51 @@
+use async_trait::async_trait;
+use tracing::instrument;
+use xtra::message_channel::MessageChannel;
+use xtra::refcount::RefCounter;
+
+/// Arbitrarily hard number allowing flexibility of prioritising messages in actors
+const INTERNAL_MSG_PRIORITY: u32 = 100;
+
+#[async_trait]
+pub trait SendNext<M: Send + 'static, R> {
+    /// Dispatch a prioritised message to an actor without waiting for it to handle it.
+    // If the actor is not connected, it logs a warning.
+    async fn send_next(&self, msg: M);
+}
+
+#[async_trait]
+impl<A, M> SendNext<M, ()> for xtra::Address<A>
+where
+    A: xtra::Handler<M, Return = ()>,
+    M: Send + 'static,
+{
+    #[instrument(skip_all)]
+    async fn send_next(&self, msg: M) {
```

Comment:
Since it's meant to be used as a prioritized message, why don't you return an error and let the caller decide?

Reply:
IMHO there is no error strategy / decision that can really be made in such context - we never did anything other than logging it (if we remembered to do so :/ ).
also, in this case, the API with an error could falsely suggest that the message got handled if it returns `Ok()`, but it's not the case (it just means that the actor got it delivered on the inbox.